.github/workflows/deploy-image.yaml

```diff
@@ -78,4 +78,33 @@ jobs:
           cache-to: type=gha,scope=prod
           target: geoserver-prod
 
+  publish_release_artifacts:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: |
+      github.actor != 'dependabot[bot]' &&
+      (!contains(github.event.comment.body, '/skiprelease') ||
+      !contains(github.event.comment.title, '[skiprelease]'))
+    needs: [deploy-image]
+    strategy:
+      matrix:
+        geoserverMajorVersion:
+          - 2
+        geoserverMinorVersion:
+          - minor: 25
+            patch: 2
+    steps:
+      - name: Get Latest Commit Hash
+        id: latest_commit_hash
+        run: echo "commit=$(git log -n 1 --pretty=format:%H)" >> $GITHUB_OUTPUT
+
+      - name: publish_release
+        id: tag_releases
+        run: |
+          gh release create v${{ matrix.geoserverMajorVersion }}.${{ matrix.geoserverMinorVersion.minor }}.${{ matrix.geoserverMinorVersion.patch }}--${{ steps.latest_commit_hash.outputs.commit }} --generate-notes --target ${{ steps.latest_commit_hash.outputs.commit }} --repo $GITHUB_REPOSITORY
```

Comment:
Ideally we should get the release notes from the comment body of the PR with something like


```
- name: Get Release Notes (if any)
      id: get_release_notes
      run: |
        release_notes=""
        if [[ -n "${{ github.event.comment.body }}" ]]; then
          # Check if comment body exists and doesn't contain "/skiprelease"
          if [[ ! $(echo "${{ github.event.comment.body }}" | grep -q '/skiprelease') ]]; then
            release_notes="${{ github.event.comment.body }}"
          fi
        fi

        # If no release notes in comment, use latest commit message
        if [[ -z "$release_notes" ]]; then
          release_notes=$(git log -n 1 --pretty=%B)  # Get full commit message
        fi

        echo "release_notes=$release_notes" >> $GITHUB_OUTPUT
```

then the release code becomes `--notes="$GITHUB_OUTPUTS.get_release_notes"`